### PR TITLE
Add DevFS for all device files

### PR DESCRIPTION
**Pull request by liqinggd**
Add DevFS for all device files

### CODE REVIEW

File: src/libos/src/fs/dev_fs/dev_sgx/mod.rs [hunk 1]
```diff
@@ -15,7 +15,58 @@ extern "C" {
 #[derive(Debug)]
 pub struct DevSgx;
 
-impl File for DevSgx {
+impl INode for DevSgx {
+    fn read_at(&self, offset: usize, buf: &mut [u8]) -> vfs::Result<usize> {
+        Ok(buf.len())
```
[the remark this answers is not in the document]
Comment (tatetian) [reply src/libos/src/fs/dev_fs/dev_sgx/mod.rs:20]: Agree with @guzongmin . It makes more sense to disable read/write on /dev/sgx.

File: src/libos/src/net/io_multiplexing/poll.rs [hunk 2]
```diff
@@ -19,6 +19,22 @@ bitflags! {
     }
 }
 
+impl PollEventFlags {
+    pub fn from_poll_status(poll_status: &crate::rcore_fs::vfs::PollStatus) -> Self {
```
Comment (tatetian) [inline src/libos/src/net/io_multiplexing/poll.rs:23]: `PollEventFlags` is in effect deprecated (it is the old event polling mechanism). And the virtual files under /dev/ is not likely to be polled. So just remove the code related to `PollEventFlags` in DevFs.

File: src/libos/src/fs/dev_fs/dev_sgx/mod.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (liqinggd) [reply src/libos/src/fs/dev_fs/dev_sgx/mod.rs]: Done

File: src/libos/src/net/io_multiplexing/poll.rs (hunk 2 above)
Comment (liqinggd) [reply src/libos/src/net/io_multiplexing/poll.rs]: Done

**Review by tatetian — APPROVED** [review]
LGTM

**Pull request merged**